copy_with_style.js

```diff
@@ -643,62 +731,88 @@ class Copy_With_Style {
 		}
 		if (this.debug) console.log("Scheduling a copy preparation ...");
 		this.button.disabled = true;
-		document.addEventListener('readystatechange', handler.bind(this));
+		document.addEventListener("readystatechange", handler.bind(this));
 	}
 
 	#schedule_observation() {
 		async function handler() {
-			if (this.debug) console.log(`Scheduled observer triggered ... triggers: ${this.triggers}, observe? ${this.observe}, preparations: ${this.#is_being_prepared}, ${this.#is_prepared}, ready state: ${document.readyState}`);
-			if (!this.#is_prepared && !this.#is_being_prepared && document.readyState === 'complete')
+			if (this.debug)
+				console.log(
+					`Scheduled observer triggered ... triggers: ${
+						this.triggers
+					}, observe? ${this.observe}, preparations: ${
+						this.#is_being_prepared
+					}, ${this.#is_prepared}, ready state: ${
+						document.readyState
+					}`
+				);
+			if (
+				!this.#is_prepared &&
+				!this.#is_being_prepared &&
+				document.readyState === "complete"
+			)
 				if (this.triggers.includes("observe") || this.observe) {
 					if (this.debug) console.log(`Starting observer`);
 					this.#observe_element();
 				}
 		}
 		if (this.debug) console.log(`Scheduling an observer ...`);
-		document.addEventListener('readystatechange', handler.bind(this));
+		document.addEventListener("readystatechange", handler.bind(this));
 	}
 
 	#observer = null;
 
 	#observe_element() {
-		this.#observer = new MutationObserver(this.#mutation_handler.bind(this));
+		this.#observer = new MutationObserver(
+			this.#mutation_handler.bind(this)
+		);
 
 		this.#observer.observe(this.element, {
 			subtree: true,
 			childList: true,
 			attributes: true,
 			attributeOldValue: true,
 			characterData: true,
-			characterDataOldValue: true
+			characterDataOldValue: true,
```

Comment:
Odd change from editor I presume. Puzzles me, are trailing commas better or customary in JS on object property lists?

Reply:
yes, when moving property sequence up and down, the trailing comma benifits